### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,13 +39,15 @@ Pour afficher des annonces de service (maintenance, formulaire, ...), on utilise
     $ npm run makeMigration migration-name
 
 #### Importer des fausses données
-Voir le fichier dans `test/seed/fake_data.js` qui va créer quelues psy, patients, et séances.
+Voir les fichiers dans `test/seed` qui vont cleaner la base puis créer quelues universités, psychologues, patients, et séances.
```
Comment (sandrica89) [inline README.md:42]:
```suggestion
Voir les fichiers dans `test/seed` qui vont cleaner la base puis créer quelques universités, psychologues, patients, et séances.
```

File: README.md [hunk 2]
```diff
@@ -39,13 +39,15 @@ Pour afficher des annonces de service (maintenance, formulaire, ...), on utilise
...
 
 Pour l'exécuter:
 
     $ npm run seed
 
 #### Upload un fichier sur Scalingo
+Attention, le fichier sera supprimer apres la déconnexion ! Si tu as besoin d'un fichier permanent sur scalingo, il faut passer par le répo git.
```
Comment (sandrica89) [inline README.md:49]:
```suggestion
Attention, le fichier sera supprimé après la déconnexion ! Si tu as besoin d'un fichier permanent sur scalingo, il faut passer par le repo git.
```

File: README.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ docker-compose up
 # to access psy workspace with toy data use this email login@beta.gouv.fr
 # to access psy workspace without data use this email empty@beta.gouv.fr
 # all available emails are listed here /test/seed/fake_data.js
-# http://localhost:8080/psychologue/login
+# http://localhost:3000/psychologue/login
```
Comment (sandrica89) [inline README.md:23]: Apply same change above (l.19)

File: README.md [hunk 4]
```diff
@@ -163,11 +164,6 @@ Started 1 cron jobs
 (...)
 ```
 
-
```
Comment (sandrica89) [inline README.md:166]: Previous section "Test du cron" (l.157) may also be deleted. Personaly I never used it, and import data is now automatically tested.

**Review by sandrica89 — CHANGES_REQUESTED** [review]
See PR comments.

Also note that we may add a section about the needed github variables for CI.

File: README.md (hunk 3 above)
Comment (desoindx) [reply README.md:23]: no, the data are available on the back so 8080 is the correct port

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**